test/jdk/java/awt/Focus/8000326/SetFocusTraversalKeysEnabledTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2013,2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2013, 2022, Oracle and/or its affiliates. All rights reserved.
```
There has to be space after the comma.